ResearchKit/Localized/en.lproj/ResearchKit.strings

```diff
@@ -276,6 +276,7 @@
 "CARDIO_OUTDOOR_INSTRUCTION_TITLE" = "Please move outdoors";
 "CARDIO_OUTDOOR_INSTRUCTION_TEXT" = "Your phone GPS indicates that you are currently indoors. Please step outdoors to before continuing.";
```

Comment:
just happened to notice the weird grammar here "...step outdoors to before continuing." delete the "to" I guess? unless there's more explanation warranted here? (or see https://sagebionetworks.jira.com/browse/IA-416)